### PR TITLE
Added missing Crafting Recipes and some Code Cleanup

### PR DESCRIPTION
**Pull request by chaos234**
Sorry that I wasn't able to simply change my changes from #715  but since I have now the time had, I have done those changes with a samll code cleanup (tested and it works). So #710 is fixed.

Screens (BetaRecipes):

Screens (StandartRecipes):

### CODE REVIEW

**Comment by chaos234** [comment]
Small question: Will this be merged? I know it is for 1.7.10 but it will be glad if the merge will come e. g. with a final build since many dev's are dropping 1.7.10 mods (support for it).

**Comment by bziemons** [comment]
Hey, sorry for the late answer.
I got some criticism about this, but I guess we could merge this after a few changes.
It should be very clear that you do not change any formatting in pull requests. Could you please revert the commits (preferably by adding a new commit and squashing commits afterwards)?
Also could you please revert 2f8f428da9d117ae198581cc09d7fbd433fa759d, as we do not want any changes in the set of registered items, triggered by just a configuration change.
Also revert be328b032b07930d77596d788aa1939778d689c5, because the function is declared as `@SneakyThrows`. Please look that up at https://projectlombok.org.

Anyway, thanks a lot for your commitment and keep the good work up :)

**Comment by chaos234** [comment]
Hello @theZorro266 and sorry for the trouble.

Since I no longer use Eclipse (I use Intellij now) it sometimes does things which I not recognize.
It will be glad if you can tell me, how to revert a commit and if this is "to bad" I think the best way
is to push a new commit with the back changes you introduced.

I haven't any knowledge with lombok. I use the normal Java knowledge for any kind of project, incl. the requirements for it to but lombok wasn'/isn't a req. so my IDE wokred as normal. If I should setup/include lombok, then a small help at this place will be fine.

I'll start reverting 2f8f428 and be328b0. 2f8f428 was a change because I thought that Items, which arn't in use shouldn't be registered to safe loadingtime and storage (icl. the IDs).

Well, I'll be in your IRC Channel so you can talk to me (Chaoschaot234).

Edit: 

> Anyway, thanks a lot for your commitment and keep the good work up :)

Thanks for the feedback :) - In my eyes each commit should get it's special comment. Means, if I have changed many files I wont push them with one single commit because the changes are sometimes differnet. So I decided to push a few times more but then with clear comment what have changed and why :)

**Comment by bziemons** [comment]
You'll need the lombok plugin for intellij (and have the annotation processing set, as it will tell you) to develop in LP. We use IntelliJ as well.
Why you should squash all commits into one is simple: You just made one change by adding a few recipes. You can split that one commit into one for each recipe, but it wont make any difference. Also sqashing would get rid of your revert commits. Please fully revert those commits though. Git has a command for that.
TL;DR: I just want to see **only** a few lines changed in the recipes in this pull reqeust.

**Comment by chaos234** [comment]
Tested and cleaned up :) - Should now be better ;)

**Review by bziemons — APPROVED** [review]
